bia_export/cli.py

```diff
@@ -131,9 +138,46 @@ def bia_image_to_export_image(image, study, use_cache=True):
         source_image_uuid=source_image_uuid,
         source_image_thumbnail_uri=source_image_thumbnail_uri,
         overlay_image_uri=overlay_image_uri,
+
         attributes=filter_image_attributes(image)
     )
 
+
+
+
+    # Refactor so titles and child uuids obtained in same API call - for loop
```

Comment:
These seems like a #TODO but without the tag at the start its less clear if it's meant to be that or a description of the current state (at a glance the code would suggest the former).